roles/mongodb_linux/tasks/main.yml

```
@@ -150,7 +150,7 @@
 # Tasks based on: https://docs.mongodb.com/manual/reference/ulimit/
 
 - name: Set pam limits (nproc and nofile)
-  community.general.pam_limits:
+  community.general.system.pam_limits:
```

Better add a comment like this:
```suggestion
  # TODO: replace with `community.general.system.pam_limits` once support for Ansible 2.9 is dropped
  community.general.system.pam_limits:
```
(same further below)

---

Didn't you say that you thought this might work on 2.9 (in the other thread)? I'll commit this as is. I've got another project still using this collection under 2.9 - https://github.com/rhysmeister/AutomatingMongoDBWithAnsible

---

Ouch, I meant `community.general.pam_limits` without the `.system.` part. Sorry for the confusion :-)

---

OK great. I'll ensure this gets tested against 2.9. Thanks again!

Rhys